Dockerfile

```diff
@@ -13,6 +13,9 @@ LABEL maintainer="Paulus Schoutsen <Paulus@PaulusSchoutsen.nl>"
 #ENV INSTALL_SSOCR no
 #ENV INSTALL_DLIB no
 #ENV INSTALL_IPERF3 no
+#ENV INSTALL_TENSORFLOW no
```

Comment:
I really don't like that we include tensorflow in the image. You should work an a solution that uses https://hub.docker.com/r/tensorflow/tensorflow/ for example.

Reply:
The only other solution would be to use a tensorflow serving image and have this component communicate with it via gRPC.  The other option is to include TensorFlow via `REQUIREMENTS` to have it auto load, and move the install portion of the object detection models to `setup_platform`.

The tradeoffs here is we would need to default to the least common denominator - the CPU.  That means users with a CUDA enabled GPU wouldn't be able to benefit from increased inference speed.  There also isn't an official wheel for alpine linux (which Hass.io uses).  The plan was to update that repo eventually to have TensorFlow pre-built into that image as well.